### PR TITLE
fix(dedicated-datacenter-backup): handle private cloud level backup

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
the backup server is at the private cloud level and this forces all datacenters to have same offer

ref: MANAGER-4635

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.routing.js [hunk 1]
```diff
@@ -43,6 +49,38 @@ export default /* @ngInject */ ($stateProvider) => {
...
+                      ...backup,
+                      datacenterId: datacenter.id,
+                    }),
+                ),
+          ),
+        ),
+      backupOffersUnderProcess: /* @ngInject */ (datacenterBackups) =>
+        some(
+          datacenterBackups,
+          (backup) => !(backup.isActive() || backup.isInactive()),
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.routing.js]: For clarity
```suggestion
          (backup) => !backup.isActive() && !backup.isInactive(),
```
Also, maybe a comment to indicate that there is another state for the backup, as it is disturbing to see:
"If I'm not active and I'm not inactive"

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/new/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -1,7 +1,10 @@
 {
   "dedicatedCloud_datacenter_backup_new_description1": "Il semblerait que vous n’avez pas activé de service de sauvegarde",
   "dedicatedCloud_datacenter_backup_new_description2": "Vous pouvez choisir entre 3 nouvelles offres pour la sauvegarde de vos machines virtuelles (VM) : <strong>Standard</strong>, <strong>Advanced</strong> ou <strong>Premium</strong>.",
+  "dedicatedCloud_datacenter_backup_new_description3": "Cette offre sera disponible par défaut pour tous les datacentres de ce private cloud.",
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/new/translations/Messages_fr_FR.json]:
```suggestion
  "dedicatedCloud_datacenter_backup_new_description3": "Cette offre sera disponible par défaut pour tous les datacentres de ce Private Cloud.",
```
Same for below sentences

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/upgrade/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -3,6 +3,7 @@
   "dedicatedCloud_datacenter_backup_upgrade_confirm": "Passer à l’offre {{ name }}",
   "dedicatedCloud_datacenter_backup_upgrade_cancel": "Annuler",
   "dedicatedCloud_datacenter_backup_upgrade_description": "Vous souhaitez changer votre offre de sauvegarde de l’<strong>offre {{ currentOffer }}</strong> à l’<strong>offre {{ selectedOffer }}</strong>.",
+  "dedicatedCloud_datacenter_backup_upgrade_warning": "Si vous avez activé la sauvegarde sur d’autres datacentres de ce PCC, cette offre leur sera également appliquée.",
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/upgrade/translations/Messages_fr_FR.json]: Maybe ? If you're speaking of Private Cloud in other translations
```suggestion
  "dedicatedCloud_datacenter_backup_upgrade_warning": "Si vous avez activé la sauvegarde sur d’autres datacentres de ce Private Cloud, cette offre leur sera également appliquée.",
```

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.routing.js (hunk 1 above)
Comment (ganeshkumar1989) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.routing.js]: Thanks for the feedback @JeremyDec . I have incorporated all your suggestions :)

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/new/translations/Messages_fr_FR.json (hunk 2 above)
Comment (ganeshkumar1989) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/new/translations/Messages_fr_FR.json]: This has been changed

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/upgrade/translations/Messages_fr_FR.json (hunk 3 above)
Comment (ganeshkumar1989) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/upgrade/translations/Messages_fr_FR.json]: This is changed :)

**Pull request merged**